images/build/createFlattenedDotNetCoreInstall.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT license.
+# --------------------------------------------------------------------------------------------
+
+set -ex
+
+declare -r DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
+
+source "$DIR/__dotNetCoreSdkVersions.sh"
+source "$DIR/__dotNetCoreRunTimeVersions.sh"
+
+splitSdksDir="/opt/dotnet/sdks"
+
+allSdksDir="/opt/dotnet/all"
+mkdir -p "$allSdksDir"
+
+# Copy latest muxer and license files
+cp -f "$splitSdksDir/3/dotnet" "$allSdksDir"
```

Comment:
@kichalla should we parameterize the folder that we're getting this content from? This could be something that's easy to overlook if/when we move to .NET Core 4, .NET Core 5, etc. and we would be copying over potentially outdated content.

Reply:
I agree. Totally valid point. Will update it.

Reply:
We could use the `lts` link

Reply:
I considered `lts` but currently lts points to `2.1` but we already have `3.0` available, so ideally we want the latest version

Reply:
@cormacpayne Can you take at this PR again? I made the change where we always get the latest version of SDK dynamically.